e2e/__snapshots__/fablo-config-hlf2-2orgs-2chaincodes-raft.yaml.test.ts.snap

```diff
@@ -947,6 +954,36 @@ services:
       - ../fabric-config/chaincode-packages:/var/hyperledger/cli/chaincode-packages/
       - \\"$CHAINCODES_BASE_DIR/./chaincodes/chaincode-kv-node/:/var/hyperledger/cli/chaincode1/\\"
       - \\"$CHAINCODES_BASE_DIR/./chaincodes/chaincode-java-simple/:/var/hyperledger/cli/chaincode2/\\"
+    depends_on:
+      - ca.org1.com
+      - peer0.org1.com
+      - peer1.org1.com
+    networks:
+      - basic
+
+  fablo-rest.org1.com:
+    container_name: fablo-rest.org1.com
+    image: softwaremill/fablo-rest:\${FABLO_REST_VERSION}
+    tty: true
```

Comment:
why o we need `tty` ?

Reply:
removed, fablo rest does not need it.

cli, however, does...